starwars/src/App.js

```diff
@@ -1,19 +1,80 @@
-import React from 'react';
+
+
+
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import People from './components/People'
 import './App.css';
 
+
+
+const AppContainer = styled.div `
+  display: flex; 
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  width: 100%;
+  height: 100%;
+`;
+
+
+
+const Button = styled.div`
+  color: red;
+`;
```

Comment:
Great job using styled components! Styling looks great. The cards being so huge took me a bit by surprise! But awesome job!!